### ISSUE TITLE
Windows 10 Fall Creators Update

### ISSUE THREAD

**Issue by dd3v** [ardevd/gitwrap#4]
Hi! GitWrap does not support last version of windows 10 =\

**Comment by ardevd** [comment]
Hi and thanks for the report. Could you please elaborate on the issue? I'm not on a Windows system these days so I have not had the chance to do much testing on the Fall Creators Update.

**Comment by dd3v** [comment]
In previous version of windows, i used gitwrap like git client for idea. Now ide return error: "Can't start GitWrap.exe Probably the path to Git executable is not valid.". 

Windows 10 (fall creators update) x64
Smart Screen: disabled

**Comment by ardevd** [comment]
Thanks. I will update my Windows VM and give it a go.

**Comment by ardevd** [comment]
Hello again. 

I've done some testing on the Fall Creators update and everything seems to work fine so far. I'll test with Android Studio tonight and see if that makes a difference. Are you sure you havent moved GitWrap.exe? The error message seemingly indicates it cant even find the executable.

**Comment by ardevd** [comment]
Update: Tested with Android Studio and I'm seeing the same thing. Not yet sure if it's a bug with IDEA, Windows 10 fall creators update or GitWrap. However, since GitWrap seems to work fine from cmd I am inclined to blame IDEA or WSL. 

Have you recently upgraded your IDE or is it just Windows you've updated?

**Comment by ardevd** [comment]
So I can rule out IDEA as older versions of Android Studio exhibits the same behavior. So this is obviously a WSL issue. I'll file a bug report.

**Comment by ardevd** [comment]
Confirmed: It's a WSL issue. [See here for more details. ](https://github.com/Microsoft/WSL/issues/2592)

TLDR: It's a known issue in WSL which has been fixed in the latest insider builds. I haven't been able to work around the issue so for now there's not much I can do.

**Comment by ardevd** [comment]
@dd3v Hello again. Could you confirm whether this issue has been fixed now?

**Comment by ardevd** [comment]
I can confirm that GitWrap works fine with the latest version of Windows 10. Closing this. Please open a new issue if you're still having problems.
